x-pack/plugins/synthetics/e2e/journeys/uptime/private_locations/add_monitor_private_location.ts

```diff
@@ -57,37 +59,34 @@ journey('AddPrivateLocationMonitor', async ({ page, params }) => {
     await uptime.waitForLoadingToFinish();
 
     await page.click('input[name="name"]');
-    await page.fill('input[name="name"]', 'Private location monitor');
-    await page.click('label:has-text("Test private location Private")');
+    await page.fill('input[name="name"]', monitorName);
+    await page.click('label:has-text("Test private location Private")', TIMEOUT_60_SEC);
     await page.selectOption('select', 'http');
     await page.click(byTestId('syntheticsUrlField'));
     await page.fill(byTestId('syntheticsUrlField'), 'https://www.google.com');
 
     await page.click('text=Save monitor');
 
-    await page.click('text=Private location monitor');
+    await page.click(`text=${monitorName}`);
 
     await page.click('[data-test-subj="superDatePickerApplyTimeButton"]');
   });
-  step('Click [placeholder="Find apps, content, and more."]', async () => {
-    await page.click('[placeholder="Find apps, content, and more."]');
-    await page.fill('[placeholder="Find apps, content, and more."]', 'integ');
-    await Promise.all([
-      page.waitForNavigation(/* { url: '${kibanaUrl}/app/integrations/browse' }*/),
-      page.click('text=Integrations'),
-    ]);
-    await page.click('text=Display beta integrations');
-    await page.click('text=Installed integrations');
-    expect(page.url()).toBe(`${kibanaUrl}/app/integrations/installed`);
-    await page.click(`text=Elastic Synthetics`);
-    await page.click('text=Integration policies');
+
+  step('Integration cannot be edited in Fleet', async () => {
+    await page.goto(`${kibanaUrl}/app/integrations/detail/synthetics/policies`);
+    await page.waitForSelector('h1:has-text("Elastic Synthetics")');
+    await page.click(`text=${monitorName}`);
+    await page.waitForSelector('h1:has-text("Edit Elastic Synthetics integration")');
+    await page.waitForSelector('text="This package policy is managed by the Synthetics app."');
   });
-  step('Click text=Edit Elastic Synthetics integration', async () => {
-    await assertText({ page, text: 'This table contains 1 rows out of 1 rows; Page 1 of 1.' });
-    await page.click('[data-test-subj="integrationNameLink"]');
-    const btn = await page.locator(byTestId('syntheticsEditMonitorButton'));
-    expect(await btn.getAttribute('href')).toBe(`/app/synthetics/edit-monitor/${monitorId}`);
```

Comment:
i think some mixup has happened here during conflict resolution

Reply:
I opted not to keep it, because basically my test actually clicks the button and ensures we are on the right page by looking for selectors specifically on that page. But I'll add it back anyway.